nucypher/cli/commands/stake.py

```diff
@@ -320,13 +320,13 @@ def config(general_config, config_file, config_options):
 @stake.command('list')
 @group_staker_options
 @option_config_file
-@click.option('--all', help="List all stakes, including unlocked", is_flag=True)
+@click.option('--show-all', help="List all stakes, including unlocked and inactive", is_flag=True)
```

Comment:
I find "show" to be a bit pedantic here and prefer simply `--all`.  There is an additional kwarg for click options to map to an alias to avoid name-shadowing the all function here.

Reply:
✔️